mudlet-mapper.xml

```diff
@@ -4963,7 +4995,48 @@ function mmp.gotoArea (where, number, dashtype, exact)
     return
   end
 
-  local getStopWatchTime, tonumber, getPath = getStopWatchTime, tonumber, mmp.getPath
+	-- add in temporary links to clouds / stratospheres to enable pathfinding, auto-repath will occur when moved to outside -- PapaGuacamole
+	local madeEast, madeNorth, madeWest
+	if mmp.game == &quot;achaea&quot; and mmp.settings.harness then			 	
+			if mmp.oncontinent(areaid, &quot;Eastern_Isles&quot;) then
```

Comment:
From what I read here, the `areaid` is the ID of the **target** area. In the code before, you used the ID of the area you were currently in. Which version is correct?